applications/ecallmgr/src/ecallmgr_fs_nodes.erl

```diff
@@ -377,6 +390,12 @@ handle_call({'is_node_up', Node}, _From, #state{nodes=Nodes}=State) ->
                {'ok', #node{connected=Connected}} -> Connected
            end,
     {'reply', Resp, State};
+handle_call({'is_node', Node}, _From, #state{nodes=Nodes}=State) ->
+    Resp = case dict:find(Node, Nodes) of
```

Comment:
Could just do `'error' =/= dict:find(Node, Nodes)` and avoid a case expression all together